apollo-spaceport/src/spaceport.rs

```diff
@@ -0,0 +1,24 @@
+//! Main entry point for CLI command to start spaceport.
+use apollo_spaceport::spaceport::ReportSpaceport;
+use clap::Parser;
+
+const DEFAULT_LISTEN: &str = "0.0.0.0:50051";
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Address to serve
+    #[clap(short, long, default_value = DEFAULT_LISTEN)]
+    address: String,
```

Comment:
:yarn: You can probably use SockAddr directly here and let clap handle the try_from

Reply:
You are correct. Nice tip, thanks!
done: a7d7a59545df5a4fbe703a38f31606a8b1d8bf62